contrib/drivers/drivers.go

```diff
@@ -129,3 +129,28 @@ func Clean(d VolumeDriver) VolumeDriver {
 
 	return d
 }
+
+type MetricDriver interface {
+	//Any initialization the volume driver does while starting.
+	Setup() error
+	//Any operation the volume driver does while stopping.
+	Unset() error
```

Comment:
I think `Teardown()` would be better.

Reply:
You mean the function name Unset() to TearDown() ?
I hope Unset() is used across hotpot as a stranded!

Reply:
You can consider it as a standard, but in the long term we would update both of them to some name like `Teardown` so that user can better understand its meaning.

Reply:
Sure will do